### DISCUSSION TITLE
Visualizing pyControl data with python

### DISCUSSION THREAD

**Discussion by flippok** [pyControl/code#109] [General]
I am having an issue with session.plot where from what I can tell the module is running properly, but no plot is generated. Below is the commands I am using to attempt to plot the data. 

import sys
sys.path.insert(1, 'C:/Users/flippok/Anaconda3/Scripts')
import session_plot as sp
sp.session_plot('C:/Users/flippok/Documents/pyControl/data/PIT_Pavlovian_Day1/Mouse1-2023-10-16-110825.txt')

Any ideas would be greatly appreciated. Thank you!

**Comment by ThomasAkam** [comment]
Hi @flippok, it may be that the plot is being generated but not displayed because matplotlib is not in iteractive mode.  Try doing:

```py
import pylab as plt
plt.ion()  # Turn on interactive mode
```

before running the session_plot function.

**Reply by flippok** [reply]
Thank you Thomas! This fixed the issue.
